### PR TITLE
Add bold to unread host/surfing request messages

### LINKED ISSUE #923: Unread request messages are not bold
Unread chats are bold, but not host requests

### PR DESCRIPTION
**Pull request by trevvorko**
Added styling to messages in both the hosting and surfing tab to be in
bold when the last message sent is unread.

Resolves #923

### CODE REVIEW

File: app/frontend/src/service/client.ts [hunk 1]
```diff
@@ -93,7 +93,7 @@ if (process.env.NODE_ENV === "development") {
     apis.pages,
     apis.references,
     apis.requests,
-    apis.search,
```
Comment (trevvorko) [inline app/frontend/src/service/client.ts]: I ran `yarn format` and it adjusted this tab here. Let me know if I should recommit without this change.

**Comment by trevvorko** [comment]
@aapeliv I can't seem to add a reviewer or labels, I think this is because I am not considered a collaborator to the repo.

File: app/frontend/src/features/messages/requests/HostRequestListItem.tsx [hunk 2]
```diff
@@ -105,7 +110,7 @@ export default function HostRequestListItem({
                 true
               )}`}
             </Typography>
-            <TextBody noWrap>
+            <TextBody noWrap className={isUnreadClass}>
```
Comment (darrenvong) [inline app/frontend/src/features/messages/requests/HostRequestListItem.tsx]: We use the `classNames` package for dealing with this kind of logic, so you can do something like this instead:

```suggestion
            <TextBody noWrap className={classNames({ [classes.unread]: isUnread })} />
```

File: app/frontend/src/features/messages/requests/HostRequestListItem.tsx [hunk 3]
```diff
@@ -48,6 +49,10 @@ export default function HostRequestListItem({
   const { data: otherUser, isLoading: isOtherUserLoading } = useUser(
     isHost ? hostRequest.fromUserId : hostRequest.toUserId
   );
+  const isUnreadClass =
+    hostRequest.lastSeenMessageId !== hostRequest.latestMessage?.messageId
+      ? classes.unread
+      : "";
```
Comment (darrenvong) [inline app/frontend/src/features/messages/requests/HostRequestListItem.tsx]:
```suggestion
  const isUnread = hostRequest.lastSeenMessageId !== hostRequest.latestMessage?.messageId;
```
When used with `classNames` suggestion below

File: app/frontend/src/service/client.ts (hunk 1 above)
Comment (darrenvong) [reply app/frontend/src/service/client.ts]: Only a small change here so it's cool to keep this in this case :)

**Review by darrenvong — CHANGES_REQUESTED** [review]
Thanks for your first contribution!

File: app/frontend/src/features/messages/requests/HostRequestListItem.tsx (hunk 2 above)
Comment (trevvorko) [reply app/frontend/src/features/messages/requests/HostRequestListItem.tsx]: This makes sense, thanks for letting me know. I made the change in the latest commit.

**Comment by darrenvong** [comment]
Can you rebase with `develop` as well please? The repo is changing fast and stuff has broken as a result of branches not being up to date with `develop` so just want to make doubly sure this will be fine.

**Comment by trevvorko** [comment]
Yeah I think I rebased correctly, let me know if there's something else I haven't done correctly, and thanks for being so helpful.

**Review by darrenvong — APPROVED** [review]
Looks good! Feel free to take a look at other issues you might be interested in working on

**Pull request merged**